platform/servicecontrol.include.md

```diff
@@ -10,6 +10,10 @@ To enable [ServiceControl](/servicecontrol) to gather this information, configur
  * configure [recoverability](/nservicebus/recoverability) to store information on messages failures;
  * [install plugins on the endpoints](/servicecontrol/plugins/) to monitor their health and sagas and use custom checks.
 
+NOTE: All endpoints should be configured to forward to the same audit, error, and ServiceControl plugin queues unless the system has been sharded between multiple ServiceControl instances.
```

Comment:
I am OK with removing the end part from this PR. I thought we talked about splitting your environment in our guidance doco, but apparently not. Since we sometimes provide guidance for large scale customers to split their environment between multiple SC installs, perhaps we should plan another PR to add that option?